README.md

```diff
@@ -45,78 +45,115 @@ Top-level:
+                     [--ctfe FILE] [--rekor-root-pubkey FILE]
+                     [--oidc-issuer URL] [--staging]
+                     FILE [FILE ...]
+
+positional arguments:
+  FILE                  The file to sign
+
+options:
+  -h, --help            show this help message and exit
+
+OpenID Connect options:
+  --identity-token TOKEN
+                        the OIDC identity token to use (default: None)
+  --oidc-client-id ID   The custom OpenID Connect client ID to use during
+                        OAuth2 (default: sigstore)
+  --oidc-client-secret SECRET
+                        The custom OpenID Connect client secret to use during
+                        OAuth2 (default: None)
   --oidc-disable-ambient-providers
-                                  Disable ambient OIDC detection (e.g. on
-                                  GitHub Actions)
-  --output-signature FILE         With a value, write a single signature to
-                                  the given file; without a value, write each
-                                  signing result to {input}.sig
-  --output-certificate FILE       With a value, write a single signing
-                                  certificate to the given file; without a
-                                  value, write each signing certificate to
-                                  {input}.cert
-  --fulcio-url URL                The Fulcio instance to use (conflicts with
-                                  --staging)  [default:
-                                  https://fulcio.sigstore.dev]
-  --rekor-url URL                 The Rekor instance to use (conflicts with
-                                  --staging)  [default:
-                                  https://rekor.sigstore.dev]
-  --ctfe FILE                     A PEM-encoded public key for the CT log
-                                  (conflicts with --staging)
-  --rekor-root-pubkey FILE        A PEM-encoded root public key for Rekor
-                                  itself (conflicts with --staging)
-  --help                          Show this message and exit.
+                        Disable ambient OpenID Connect credential detection
+                        (e.g. on GitHub Actions) (default: False)
+
+Output options:
+  --output              Write signature and certificate results to default
+                        files ({input}.sig and {input}.crt) (default: False)
+  --output-signature FILE
+                        Write a single signature to the given file; conflicts
+                        with --output and does not work with multiple input
+                        files (default: None)
+  --output-certificate FILE
+                        Write a single certificate to the given file;
+                        conflicts with --output and does not work with
+                        multiple input files (default: None)
+  --force               Overwrite preexisting signature and certificate
```

Comment:
"force" seems a little too strong here on second thought, makes me think of force-pushing. Perhaps "overwrite" would be better?